index.js

```diff
@@ -227,6 +227,26 @@ function browse(options) {
   });
 }
 
+function boards(options) {
+  logger = log.getInstance(options.verbose);
+
+  getBaseBranchName().then(function (curBranchName) {
+    getRemoteForBranch(curBranchName).then(function (remote) {
+      if (!remote) {
+        console.error(colors.red('Branch ' + curBranchName + ' is not tracked by any remote branch.'));
+        console.log('Set the remote tracking by `git remote git branch --set-upstream <branch-name> <remote-name>/<branch-name>`');
+        console.log('Eg: `git branch --set-upstream foo upstream/foo`');
```

Comment:
Shouldn't there be `process.exit(1)` here. You will want to stop the process with error when getting the remote has failed.

Reply:
I just adopted the implementation of `browse` command. I can add a separate pull request to fix it in both commands, if this one will be accepted.